Exoft.Gamification.Api.Data/Repositories/UserRepository.cs

```diff
@@ -34,7 +28,8 @@ protected override IQueryable<User> IncludeAll()
                 .Include(s => s.Roles)
                     .ThenInclude(s => s.Role)
                 .Include(s => s.Achievements)
-                    .ThenInclude(s => s.Achievement);
+                    .ThenInclude(s => s.Achievement)
+                .OrderByDescending(i => i.XP);
```

Comment:
Again ordering... As possible solution it could be abstract method on base class that you have to override in child classes